src/main/java/com/aws/greengrass/deployment/DeploymentStatusKeeper.java

```diff
@@ -80,6 +80,8 @@ public void persistAndPublishDeploymentStatus(String deploymentId, DeploymentTyp
             Topics processedDeployments = getProcessedDeployments();
             Topics thisJob = processedDeployments.createInteriorChild(String.valueOf(System.currentTimeMillis()));
             thisJob.replaceAndWait(deploymentDetails);
+            logger.atDebug().kv(DEPLOYMENT_ID_KEY_NAME, deploymentId).kv(DEPLOYMENT_STATUS_KEY_NAME, status)
```

Comment:
this needs to be info, right? We don't want to need to increase log level to get such basic information IMO.